modules/adyoulikeBidAdapter.js

```diff
@@ -122,8 +121,8 @@ export const spec = {
       payload.ortb2 = bidderRequest.ortb2;
     }
 
-    if (deepAccess(bidderRequest, 'userId')) {
-      payload.userId = createEidsArray(bidderRequest.userId);
+    if (deepAccess(bidderRequest, 'userIdAsEids')) {
```

Comment:
@guiann FYI - I believe this is currently broken and you're never getting `userId` from your adapter. This is because `userId` / `userIdAsEids` are *not* properties of `bidderRequest`, but of individual bidRequest objects (`bidRequests[].userIdAsEids` in this scope).

Reply:
Thank you, we'll have a look and fix it in a future PR.